### DISCUSSION TITLE
Qt 6.5.1 bottle contains static versions of the Qt Permissions plugins

### DISCUSSION THREAD

**Discussion by vicr123** [Homebrew/discussions#4776] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: 12c8778af976269344f3c0b914243e178fc4ffbd
Last commit: 2 days ago
Core tap JSON: 08 Sep 09:57 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Applications/Xcode-14.3.1.app/Contents/Developer/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1 => /Applications/Xcode-14.3.1.app/Contents/Developer
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /opt/homebrew/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/opt/homebrew/sbin:$PATH"' >> ~/.zshrc
```

### Description of issue

After installing the Qt bottle, I started writing an app that uses the new Qt Permissions API, and it seems to be having issues loading a permissions plugin.

Upon further inspection, I viewed the `/opt/homebrew/Cellar/qt/6.5.1_3/share/qt/plugins/permissions` directory and noticed that all of the plugins in that folder were static versions, not dynamic libraries.

<img width="255" alt="image" src="https://github.com/Homebrew/discussions/assets/7380161/8182015c-2950-4a39-86ad-669584a5b867">

Is this an issue with the recipe, or is it an upstream issue I should report at Qt?

**Comment by Bo98** [comment, marked as answer]
This appears to be intentional by upstream: https://github.com/qt/qtbase/blob/v6.5.1/cmake/QtPluginHelpers.cmake#L536
